### PR TITLE
Telegraf tag

### PR DESCRIPTION
**Pull request by senkrish**
Bump version for config RPMs.
Use git tag to extract the version for the RPMs.

### CODE REVIEW

File: Arista/arista-package.sh [hunk 1]
```diff
@@ -117,43 +119,44 @@ mkdir -p $TMP_CONFIG_DIR/etc/telegraf/telegraf.d
...
 rm -f $TMP_CONFIG_DIR/etc/telegraf/telegraf.d/*
 cp $CONFIG_FILES_DIR/telegraf-redis.conf $TMP_CONFIG_DIR/etc/telegraf/telegraf.d/
-fpm -s dir -t rpm $CONFIG_FPM_ARGS --iteration "$CONFIG_FILES_ITER" -v "$REDIS_CONFIG_FILES_VER" --description "$DESCRIPTION" -n "telegraf-Redis" etc lib || cleanup_exit 1
+fpm -s dir -t rpm $CONFIG_FPM_ARGS --description "$DESCRIPTION" -n "telegraf-Redis" etc lib || cleanup_exit 1
 
 # Perforce-Config
 rm -rf $TMP_CONFIG_DIR/etc/telegraf/telegraf.d/*
 cp $CONFIG_FILES_DIR/telegraf-perforce.conf $TMP_CONFIG_DIR/etc/telegraf/telegraf.d/
-fpm -s dir -t rpm $CONFIG_FPM_ARGS --iteration "$CONFIG_FILES_ITER" -v "$PERFORCE_CONFIG_FILES_VER" --description "$DESCRIPTION" -n "telegraf-Perforce" etc lib || cleanup_exit 1
+echo fpm -s dir -t rpm $CONFIG_FPM_ARGS --description "$DESCRIPTION" -n "telegraf-Perforce" etc lib || cleanup_exit 1
```
Comment (manojm321) [inline Arista/arista-package.sh]: may be forgot to delete this?

**Review by manojm321 — COMMENTED** [review]
Thanks for making rpm versioning easier.

**Review by manojm321 — COMMENTED** [review]
Changes LGTM.

I see that telegraf-linux.conf is not updated to export smart data. Is this intentional?

**Comment by senkrish** [comment]
telegraf-linux.conf already has the change to export smart data, from an earlier review.

**Review by manojm321 — APPROVED** [review]
I see, my local repo was few commits behind.

**Pull request merged**